src/ar/azora/src/eu/kanade/tachiyomi/extension/ar/azora/Azora.kt

```diff
@@ -8,6 +8,7 @@ class Azora : Madara("Azora", "https://azoramoon.com", "ar") {
     override val mangaSubString = "series"
     override val useNewChapterEndpoint = false
     override fun chapterListSelector() = "li.wp-manga-chapter:not(.premium-block)" // Filter fake chapters
+    override val mangaDetailsSelectorDescription = "div.tab-summary > div.summary_content_wrap > div > div.post-content > div.manga-summary"
```

Comment:
That selector is very specific and will likely break in the future.

Consider:
```suggestion
    override val mangaDetailsSelectorDescription = ".manga-summary"
```

Reply:
Fix it